### PR TITLE
Symptom checker screen results ui

### LINKED ISSUE #1108: Symptom Checker Screen: Results UI
**The product definition, design, content, and assets are a WIP and are neither finalized nor fully approved by WHO. The purpose of this task is to prototype the feature so we can execute quickly when the non-engineering deliverables are finalized.**

### PR DESCRIPTION
**Pull request by jasonTelanoff**
Creates Symptom Checker Results Page

Closes #1108 

Screenshot_20200507-165412

#### How did you test the change?
* [ ] iOS Simulator
* [ ] Android Emulator
* [ ] iOS Device
* [x] Android Device
* [ ] `curl` to a dev App Engine server

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      title: "Check-Up",
+      body: <Widget>[_buildBody(context)],
+    );
+  }
+
+  Widget _buildBody(BuildContext context) => SliverList(
+          delegate: SliverChildListDelegate([
+        SafeArea(child: bodyWidget(context)),
+        videoWidget(context),
+        floorWidget(context)
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:29]:
```suggestion
        floorWidget(context),
```
add trailing comma

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      disableBackButton: true,
+      showHeader: true,
+      title: "Check-Up",
+      body: <Widget>[_buildBody(context)],
+    );
+  }
+
+  Widget _buildBody(BuildContext context) => SliverList(
+          delegate: SliverChildListDelegate([
+        SafeArea(child: bodyWidget(context)),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:27]:
```suggestion
        SafeArea(
          child: bodyWidget(context),
        ),
```

trailing comma

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          // Go to video
+        },
+        crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisAlignment: MainAxisAlignment.center,
+        verticalPadding: 0,
+        borderRadius: 50,
+        titleStyle: TextStyle(
+            color: CupertinoColors.white,
+            fontSize: 18,
+            fontWeight: FontWeight.w600),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:77]:
```suggestion
            fontWeight: FontWeight.w600,
       ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        videoWidget(context),
+        floorWidget(context)
+      ]));
+
+  Widget bodyWidget(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.only(top: 64, left: 24, right: 24),
+      child: Column(
+        children: <Widget>[
+          Text(
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:37]: Use ThemedText

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                          color: Constants.backgroundColor,
+                          fontSize: 16,
+                          fontWeight: FontWeight.bold),
+                    )
+                  : Text(
+                      'It is likely you have contracted the Coronavirus.',
+                      style: TextStyle(
+                          color: Constants.backgroundColor,
+                          fontSize: 16,
+                          fontWeight: FontWeight.bold),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:108]:
```suggestion
                          fontWeight: FontWeight.bold,
                      ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+          padding: EdgeInsets.only(top: 47, bottom: 42, left: 34, right: 18),
+          child: Column(
+            children: <Widget>[
+              risk < 2
+                  ? Text(
+                      'It is unlikely you have contracted the Coronavirus.',
+                      style: TextStyle(
+                          color: Constants.backgroundColor,
+                          fontSize: 16,
+                          fontWeight: FontWeight.bold),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:101]:
```suggestion
                          fontWeight: FontWeight.bold,
                      ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+                    ),
+              SizedBox(height: 12,),
+              Text(
+                'Stay aware of the latest information on the COVID-19 outbreak, available on the WHO website and through your national and local public health authority. Most people who become infected experience mild illness and recover, but it can be more severe for others.',
+                style: TextStyle(
+                  color: Constants.backgroundColor,
+                  fontSize: 16,
+                ),
+              )
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:117]:
```suggestion
              ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+                      fontWeight: FontWeight.w900,
+                      color: Constants.primaryColor,
+                      fontSize: 48),
+                )
+              : Text(
+                  'Unlikely',
+                  style: TextStyle(
+                      fontWeight: FontWeight.w900,
+                      color: Constants.primaryColor,
+                      fontSize: 48),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:54]:
```suggestion
                      fontSize: 48,
                  ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+            'Your results',
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 18),
+          ),
+          risk < 2
+              ? Text(
+                  'Unlikely',
+                  style: TextStyle(
+                      fontWeight: FontWeight.w900,
+                      color: Constants.primaryColor,
+                      fontSize: 48),
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:47]:
```suggestion
                      fontSize: 48,
                  ),
```

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+  Widget bodyWidget(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.only(top: 64, left: 24, right: 24),
+      child: Column(
+        children: <Widget>[
+          Text(
+            'Your results',
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 18),
+          ),
+          risk < 2
```
Comment (theswerd) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:41]: Should this be a content bundle?

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart (hunk 6 above)
Comment (jasonTelanoff) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart]: This was fixed in latest commit

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart (hunk 10 above)
Comment (jasonTelanoff) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart:43]: No clue

File: client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart (hunk 8 above)
Comment (jasonTelanoff) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart]: Outdated

**Comment by advayDev1** [comment]
The UI has completely changed.  We will need to revise #1236 after it goes in however.

**Pull request closed without merging**